conformance/tests/enums_behaviors.py

```diff
@@ -23,8 +23,13 @@ class Color(Enum):
 # > constructor. Instead, the call performs a value-based lookup of an
 # > enum member.
 
-assert_type(Color["RED"], Color)  # 'Literal[Color.RED]' is also acceptable
```

Comment:
felt odd to assert_type here and say that another type is also acceptable. I tried to put both versions down, so that type checkers have discretion on which one to implement w/o the test failing.

I assume that if a checker emits an error on both lines the test will fail - could the maintainers please confirm this assumption?

Reply:
That's right; this is documented in the "Test case syntax" section in conformance/README.